web/src/features/hfiCalculator/pages/HfiCalculatorPage.tsx

```diff
@@ -1,22 +1,79 @@
-import React, { useEffect } from 'react'
+import React, { useEffect, useState } from 'react'
 
 import { Container, PageHeader, PageTitle } from 'components'
 import DailyViewTable from 'features/hfiCalculator/components/DailyViewTable'
-import { useDispatch } from 'react-redux'
 import { fetchHFIStations } from '../slices/stationsSlice'
+import { fetchHFIDailies } from '../slices/hfiCalculatorSlice'
+import { useDispatch, useSelector } from 'react-redux'
+import { DateTime } from 'luxon'
+import {
+  selectHFIDailies,
+  selectHFIStations,
+  selectHFIStationsLoading
+} from 'app/rootReducer'
+import { makeStyles, CircularProgress } from '@material-ui/core'
+import { StationDaily } from 'api/hfiCalculatorAPI'
 
 const HfiCalculatorPage: React.FunctionComponent = () => {
+  const useStyles = makeStyles({
+    container: {
+      display: 'flex',
+      justifyContent: 'center'
+    }
+  })
+  const classes = useStyles()
+
   const dispatch = useDispatch()
+  const { dailies, loading } = useSelector(selectHFIDailies)
+  const { fireCentres } = useSelector(selectHFIStations)
+  const stationDataLoading = useSelector(selectHFIStationsLoading)
+  const [currentDay, setCurrentDay] = useState(DateTime.now())
+
   useEffect(() => {
+    const startTime = currentDay
+      .startOf('day')
+      .toUTC()
+      .valueOf()
+    const endTime = currentDay
+      .endOf('day')
+      .toUTC()
+      .valueOf()
+    dispatch(fetchHFIDailies(startTime, endTime))
     dispatch(fetchHFIStations())
```

Comment:
don't we need to first fetch the HFI stations to get the station codes before we can query WFWX with the weather date for those stations?

Reply:
This PR assumes you're interested in all stations regardless -- it leaves the station code params in the request empty.

When we do unselecting/selecting this will have to be addressed.